cleverpush/src/main/java/com/cleverpush/CleverPush.java

```diff
@@ -1073,6 +1105,12 @@ private void subscribe(boolean newSubscription, SubscribedCallbackListener subsc
             return;
         }
 
+        if (Build.VERSION.SDK_INT >= Constants.ANDROID_13_VERSION) {
```

Comment:
Add `&& !this.hasNotificationPermission()`

Reply:
Okay i added